server_installation/topics/operator/monitoring-stack.adoc

```diff
@@ -0,0 +1,31 @@
+
+=== Monitoring Stack
+
+{project_operator} integrates with {application_monitoring_operator} providing Graphana Dashboard and Prometheus Alerts. The integration is seamless and doesn't require any action from the user. However, {application_monitoring_operator} needs to be properly installed in the cluster.
+
+==== Installing {application_monitoring_operator}
+
+Prior to running {project_operator} it is recommended to install {application_monitoring_operator} by following the link:{application_monitoring_operator_installation_link}[documentation]. After {application_monitoring_operator} is up and running, you need to annotate a namespace used for {project_operator} installation. Here's an example:
```

Comment:
`you need to annotate the namespace used for ...`?

Reply:
Fixed.